code/modules/jobs/job_types/curator.dm

```diff
@@ -47,14 +47,30 @@
 		/obj/item/choice_beacon/hero = 1,
 	)
 	belt = /obj/item/modular_computer/pda/curator
-	ears = /obj/item/radio/headset/headset_srv
+	ears = /obj/item/radio/headset/headset_srvent
 	shoes = /obj/item/clothing/shoes/laceup
 	l_pocket = /obj/item/laser_pointer/green
 	r_pocket = /obj/item/key/displaycase
 	l_hand = /obj/item/storage/bag/books
 
 	accessory = /obj/item/clothing/accessory/pocketprotector/full
 
+/datum/outfit/job/curator/pre_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(visualsOnly)
+		return ..()
+
+	/// There can be only one cameraman on this station, and no, not that kind
+	var/static/cameraman_choosen = FALSE
+	if(!cameraman_choosen)
+		backpack_contents = list(
+			/obj/item/barcodescanner = 1,
+			/obj/item/choice_beacon/hero = 1,
+			/obj/item/broadcast_camera = 1,
+		)
```

Comment:
```suggestion
		backpack_contents[/obj/item/broadcast_camera] = 1
```
This way the default backpack contents can be modified without this needing to be modified too.